samples/file-encryptor/enclave/mbedtls_src/encryptor.cpp

```diff
@@ -41,15 +46,23 @@ int ecall_dispatcher::initialize(
     }
 
     // initialize aes context
-    mbedtls_aes_init(&m_aescontext);
+    m_aescontext = (struct aes_context*) malloc(sizeof(struct aes_context));
+    if (m_aescontext == NULL)
```

Comment:
since this file is already C++ specific (e.g., using std::string) then this should use the C++ keyword nullptr instead of NULL.

Reply:
changed instances of NULL to nullptr.